examples/syscall_steal.c

```diff
@@ -61,7 +61,7 @@ module_param(sym, ulong, 0644);
 
 #endif /* Version < v5.7 */
 
-static unsigned long **sys_call_table;
+static unsigned long **_sys_call_table;
```

Comment:
`_sys_call_table` is not a good name, if we do want to avoid the compiler warning we should have a better name not just add `_` or `__` prefix.

Reply:
Shall we call it `stealed_sys_call_table`?😂

Reply:
And it is actually an error rather than a warning. I am on gcc 12.2.0.

Reply:
> And it is actually an error rather than a warning. I am on gcc 12.2.0.

Sorry for my incorrect words.

> Shall we call it sys_call_table_stealed?😂

It is better than the `_sys_call_table` actually. LOL

Reply:
I edited my comment and I think `sys_call_table_acquired` might be better.

Reply:
But anyway it IS stolen. Let's still make it sys_call_table_stolen.